_posts/2021-07-16-release-0.9-future.md

```diff
@@ -0,0 +1,39 @@
+
+  - [gfx-rs](https://github.com/gfx-rs/gfx) makes low-level GPU programming portable with low overhead. It's a single Vulkan-like Rust API with multiple backends that implement it: Direct3D 12/11, Metal, Vulkan, and even OpenGL ES.
+  - [naga](https://github.com/gfx-rs/naga) translates the shaders between languages, including WGSL. Also provides validation and processing facilities on the intermediate representation.
+  - [wgpu](https://github.com/gfx-rs/wgpu) is built on top of gfx-rs and [gpu-alloc](https://github.com/zakarumych/gpu-alloc)/[gpu-descriptor](https://github.com/zakarumych/gpu-descriptor). It provides safety, accessibility, and strong portability of applications.
+
+Following our release cadence every few months, we rolled out the 0.9 version through all of the gfx projects!
+See [gfx-rs changelog](https://github.com/gfx-rs/gfx/blob/master/CHANGELOG.md#hal-090-18-06-2021), [wgpu changelog](https://github.com/gfx-rs/wgpu/blob/master/CHANGELOG.md#v09-2021-06-18), and [naga changelog](https://github.com/gfx-rs/naga/blob/master/CHANGELOG.md#v05-2021-06-18) for the details.
+
+`naga` has matured signifgantly since the last release. `wgsl` parsing has imporved incredibly, targetting an up-to-date spec. `spirv` parsing support has had numerous bugs fixed, and `glsl` support is starting to take shape, though still in an alpha state.
+
+`wgpu` validation has continued to improve. Many validation holes were plugged with the last release. Through the combined work in `wgpu` and `naga`, validation holes have been sured up.
+
+`wgpu` performance is still a vital target for us, so we have done work on improving the overhead at resource tracking. We've reduced unnecisary overhead through only doing stateful tracking for resources that have complex states. These changes were made from benchmarks of Gecko's WebGPU implemention which showed that tracking was a bottleneck. You can read more about it [#1413](https://github.com/gfx-rs/wgpu/issues/1413).
+
+## wgpu family reunion, relicense, and the future
+
+`wgpu` has had a number of large internal changes which are laying the future for wgpu to be a safe, efficient, and portable api for doing cross-platform graphics.
+
+`wgpu` has been relicensed from `MPL-2.0` to `MIT/Apache-2.0`. Thank you to all 142 people who replied to the issue and made this happen. This relicense is an important change because it allows the posibility of adding backends that target APIs that are behind NDAs.
+
+Thanks to feedback from our friends at the bevy game engine, we launched a initiative to make wgpu easier to contribute to. We moved wgpu-rs back into the `wgpu` repo. This means that PRs that touch both the core crate and the rust bindings no longer need multiple PRs that need to be synchronized. We have already heard from collaborators how much easier the contribution is now that there is less coordination to do. [Read more about the family reunion](https://github.com/gfx-rs/wgpu/milestone/9)
```

Comment:
the family reunion link - let's point to the closed issues list https://github.com/gfx-rs/wgpu/milestone/9?closed=1